exe/Parser/Text.hs

```diff
@@ -527,33 +527,50 @@ moduleToFileName = map (\x -> if x == '.' then '/' else x)
 dirFromFileName :: String -> String
 dirFromFileName = reverse . dropWhile (/= '/') . reverse
 
-generateModule ::
+type GeneratorRecipe a = (String, String -> Fold IO a String)
```

Comment:
Its usually helpful to read code if we write what these two things in the tuple semantically are or what GeneratorRecipe means.